.github/workflows/ci.yml

```diff
@@ -14,15 +14,17 @@ jobs:
           apt update
           apt install -y sudo
           sudo apt install -y git
-          sudo DEBIAN_FRONTEND=noninteractive apt install --yes npm
-      - name: Install commitlint
-        run: npm install
-      - name: Print versions
+          
+          sudo apt install -y curl
+          # can't install ubuntu's default nodejs version because we would get this error:
+          # error @commitlint/config-conventional@17.1.0: The engine "node" is incompatible with this module. Expected version ">=v14". Got "12.22.9"
+          curl -sL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+          sudo DEBIAN_FRONTEND=noninteractive apt install -y nodejs
+
+      - name: Install conventions dependencies
         run: |
-          git --version
-          node --version
-          npm --version
-          npx commitlint --version
+          npm install -g typescript@4.8.4
```

Comment:
@realmarv what happens if we use 4.9.4 instead of 4.8.4?

Reply:
It broke the CI

Reply:
then you didn't really fix the typescript problem, you managed to upgrade commitlint without upgrading typescript :) (which is kinda ok, for now, just rename the PR to make it clear)

Reply:
Not sure. You said it's ok for now not totally, so I thought we should upgrade the typescript as well